src/Validation/Validation.php

```diff
@@ -522,15 +522,53 @@ public static function localizedTime($check, $type = 'datetime', $format = null)
 
     /**
      * Boolean validation, determines if value passed is a boolean integer or true/false.
+     * The list of what is considered to be boolish values, may be set via $booleanValues.
      *
      * @param string $check a valid boolean
+     * @param string $booleanValues list of valid boolish values, defaults to `[true, false, 0, 1, '0', '1']`.
      * @return bool Success
      */
-    public static function boolean($check)
+    public static function boolean($check, array $booleanValues = [])
     {
-        $booleanList = [0, 1, '0', '1', true, false];
+        if (!$booleanValues) {
+            $booleanValues = [true, false, 0, 1, '0', '1'];
+        }
+
+        return in_array($check, $booleanValues, true);
+    }
+
+    /**
+     * Truthy validation, determines if value passed is truthy.
+     * The list of what is considered to be truthy values, may be set via $truthyValues.
+     *
+     * @param string $check a valid boolean
+     * @param array $truthyValues list of valid truthy values, defaults to `[true, 1, '1']`.
+     * @return bool Success
+     */
+    public static function truthy($check, array $truthyValues = [])
+    {
+        if (!$truthyValues) {
+            $truthyValues = [true, 1, '1'];
+        }
+
+        return in_array($check, $truthyValues, true);
+    }
+
+    /**
+     * Truthy validation, determines if value passed is falsey.
+     * The list of what is considered to be truthy values, may be set via $falseyValues.
+     *
+     * @param string $check a valid boolean
+     * @param array $falseyValues list of valid falsey values, defaults to `[false, 0, '0']`.
+     * @return bool Success
+     */
+    public static function falsey($check, array $falseyValues = [])
```

Comment:
Unfortunately, that is true. So for consistency this is probably ok to keep this way.